modules/simple_forms_api/lib/tasks/archive_download.rake

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+include 'simple_forms_api/s3/submission_archive_builder'
+include 'common/file_helpers'
+
+namespace :simple_forms_api do
+  desc 'Build submission archives for a collection of ' \
+       'benefits_intake_uuids, zips them, and downloads ' \
+       'them to a specified location'
+
+  task :download_archives, %i[benefits_intake_uuids download_path] => :environment do |_, args|
+    benefits_intake_uuids = args[:benefits_intake_uuids].split
+    download_path = args[:download_path] || raise('Please provide a download_path')
+
+    if benefits_intake_uuids.empty?
+      puts 'No benefits_intake_uuids provided. Please provide a collection of benefits_intake_uuids.'
+      exit 1
+    end
+
+    benefits_intake_uuids.each do |uuid|
+      archive_builder = SimpleFormsApi::S3::SubmissionArchiveBuilder.new(benefits_intake_uuid: uuid)
+      archive_dir = archive_builder.run
```

Comment:
I'm nervous about saving PDFs to our prod instances. Can we get this checked by a devops person? For example, if the pod goes down mid-operation what happens? I don't really know the first thing about how the pods are connected to a filesystem and what kind of persistence we have there.

Reply:
If one goes down mid-operation, we would lose the temporary directory that had been created for that specific uuid/submission. Given that the full build/zip/download/delete cycle happens for each submission at a time, if the process is interrupted we should only have to re-run for whichever submission we don't have downloaded to our host machine. I'm fine having someone from DevOps take a look though!

Reply:
since the operation does not require persistent storage,  I would go with @pennja recommendation to make it a retriable operation.    That will ensure the operation survives events like pod going down or host reboots which by the way clears everything in the /tmp directory.